model/textparse/openmetricsparse.go

```diff
@@ -94,14 +94,49 @@ type OpenMetricsParser struct {
 	exemplarVal   float64
 	exemplarTs    int64
 	hasExemplarTs bool
+
+	// lines ending with _created are skipped by default as they are
+	// parsed by the CreatedTimestamp method. The skipCT flag is set to
+	// false when the CreatedTimestamp method is called.
+	skipCTSeries bool
+}
+type openMetricsParserOptions struct {
+	// SkipCT skips the parsing of _created lines.
+	SkipCTSeries bool
+}
+
+type OpenMetricsOption func(*openMetricsParserOptions)
+
+func WithOMParserCTSeriesSkipped(skipCTSeries bool) OpenMetricsOption {
```

Comment:
```suggestion
// WithOMParserCTSeriesSkipped turns off exposing _created lines
// as series, which makes those only used for parsing created timestamp
// for `CreatedTimestamp` method purposes.
//
// It's recommended to use this option to avoid using _created lines for other
// purposes than created timestamp, but leave false by default for the
// best-effort compatibility.
func WithOMParserCTSeriesSkipped() OpenMetricsOption {
```

Reply:
(no need for bool setting, user can only really set it to true, no?